src/Qir/qir-utils.ps1

```diff
@@ -192,9 +192,10 @@ function Build-CMakeProject {
         $env:CXX = "clang++.exe"
         $env:RC = "clang++.exe"
 
-        if (!(Get-Command clang -ErrorAction SilentlyContinue) -and (choco find --idonly -l llvm) -contains "llvm") {
+        if ((!(Get-Command clang -ErrorAction SilentlyContinue) -and (choco find --idonly -l llvm) -contains "llvm") -or `
+            (Test-Path Env:AGENT_OS)) {
             # LLVM was installed by Chocolatey, so add the install location to the path.
-            $env:PATH += ";$($env:SystemDrive)\Program Files\LLVM\bin"
+            $env:PATH = "$($env:SystemDrive)\Program Files\LLVM\bin;$env:Path"
```

Comment:
If you have time and desire (to unify the heritage):

In one file the fragment was 
`C:\Program Files`
in the other 
`$($env:SystemDrive)\Program Files`.
It probably makes sense to use identical in both.